### PR TITLE
remove me as maintainer

### PR DESCRIPTION
**Pull request by jD91mZM2**
###### Motivation for this change

I don't use these tools anymore, so it makes sense I shouldn't have an opinion on PRs that change/update them. I know it's always unfortunate losing a reviewer, but I'm not very active anymore anyway, unfortunately. Apologies.

From now on, I'm trying not to add too many packages into nixpkgs, since flakes are available. I guess when I first started using Nix I got overexcited by how easy it was to contribute, so I added things for the sake of adding things (not because I necessarily used them).

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/scaff/default.nix [hunk 1]
```diff
@@ -1,26 +1,27 @@
 { lib, rustPlatform, fetchFromGitLab, pkg-config, openssl }:
 
-rustPlatform.buildRustPackage rec {
-  pname = "scaff";
-  version = "0.1.2";
+builtins.trace "scaff is deprecated - replaced by https://gitlab.com/jD91mZM2/inc (not in nixpkgs yet)" (
```
Comment (veprbl) [inline pkgs/development/tools/scaff/default.nix:3]: Eval warnings of this kind are not currently accepted in nixpkgs

File: pkgs/applications/virtualization/x11docker/default.nix [hunk 2]
```diff
@@ -23,7 +23,6 @@ stdenv.mkDerivation rec {
     description = "Run graphical applications with Docker";
     homepage = "https://github.com/mviereck/x11docker";
     license = lib.licenses.mit;
-    maintainers = with lib.maintainers; [ jD91mZM2 ];
     platforms = lib.platforms.linux;
```
Comment (SuperSandro2000) [inline pkgs/applications/virtualization/x11docker/default.nix:26]:
```suggestion
    maintainers = with lib.maintainers; [ ];
    platforms = lib.platforms.linux;
```

File: pkgs/desktops/xfce/panel-plugins/xfce4-vala-panel-appmenu-plugin/appmenu-gtk-module.nix [hunk 3]
```diff
@@ -26,6 +26,5 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Port of the Unity GTK Module";
     license = licenses.lgpl3;
```
Comment (SuperSandro2000) [inline pkgs/desktops/xfce/panel-plugins/xfce4-vala-panel-appmenu-plugin/appmenu-gtk-module.nix:28]:
```suggestion
    license = licenses.lgpl3;
    maintainers = with lib.maintainers; [ ];
```

File: pkgs/desktops/xfce/panel-plugins/xfce4-vala-panel-appmenu-plugin/default.nix [hunk 4]
```diff
@@ -50,7 +50,6 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Global Menu applet for XFCE4";
     license = licenses.lgpl3;
-    maintainers = with maintainers; [ jD91mZM2 ];
     broken = true;
```
Comment (SuperSandro2000) [inline pkgs/desktops/xfce/panel-plugins/xfce4-vala-panel-appmenu-plugin/default.nix:53]: If the package is broken and maintainerless we can remove it all together.

File: pkgs/tools/X11/nx-libs/default.nix [hunk 5]
```diff
@@ -42,7 +42,6 @@ stdenv.mkDerivation rec {
     description = "NX X server based on Xnest";
     homepage = "https://github.com/ArcticaProject/nx-libs";
     license = lib.licenses.gpl2;
```
Comment (SuperSandro2000) [inline pkgs/tools/X11/nx-libs/default.nix:44]:
```suggestion
    license = lib.licenses.gpl2;
    maintainers = with lib.maintainers; [ ];
```

File: pkgs/development/tools/scaff/default.nix (hunk 1 above)
Comment (jD91mZM2) [reply pkgs/development/tools/scaff/default.nix:3]: What's the best way to deprecate a package?

Comment (SuperSandro2000) [reply pkgs/development/tools/scaff/default.nix]: Remove it and add an alias to aliases.nix.

**Comment by evils** [comment]
was `pkgs/development/tools/scaff` left in intentionally?

**Comment by jD91mZM2** [comment]
You're right @evils, it wasn't. My bad!

**Pull request merged**